opts/network.go

```diff
@@ -8,9 +8,11 @@ import (
 )
 
 const (
-	networkOptName  = "name"
-	networkOptAlias = "alias"
-	driverOpt       = "driver-opt"
+	networkOptName        = "name"
+	networkOptAlias       = "alias"
+	networkOptIPv4Address = "ip"
+	networkOptIPv6Address = "ip6"
```

Comment:
Let me know if this naming LGTY; I see the compose-file uses https://docs.docker.com/compose/compose-file/#ipv4_address-ipv6_address

```yaml
        ipv4_address: 172.16.238.10
        ipv6_address: 2001:3984:3989::10
```

Reply:
naming LGTM 👍